terraform-provider.json

```diff
@@ -1,15 +1,24 @@
 {
-    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-    "description": "Upgrade all terraform providers with minor and patch upgrades",
-    "groupName": "Terraform provider minor and patch upgrades",
-    "groupSlug": "terraform-provider-minor-patch",
-    "postUpgradeTasks": {
-      "commands": [
-        "terraform-docs markdown {{{ packageFileDir }}} --hide resources,data-sources --output-file README.md --output-mode replace"
-      ],
-      "fileFilters": [
-        "**/README.md"
-      ],
-      "executionMode": "update"
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Upgrade all terraform providers with minor and patch upgrades",
+  "packageRules": [
+    {
+      "matchDatasources": ["terraform-provider"]
```

Comment:
I'm not sure this package rule does anything. What's the purpose? To group PRs matching this datasource? If yes, I think we would need to specify a group name, but I'm not sure